tests/unit-tests/config/sequelize/sequelizeConnector-test.js

```diff
@@ -0,0 +1,126 @@
+'use strict';
+const sinon = require('sinon');
+const rewire = require('rewire');
+const should = require('should');
+let databaseConfig = require('../../../../src/config/databaseConfig');
+const sequelizeConnector = rewire('../../../../src/configManager/models/database/sequelize/sequelizeConnector');
+
+describe('Cassandra client tests', function () {
+    let sandbox,
+        sequelizeModelStub,
+        sequelizeUpsertStub,
+        sequelizeStub,
+        sequelizeDefineStub,
+        sequelizeGeValueetStub,
+        sequelizeGetStub;
+
+    before(() => {
+        sandbox = sinon.sandbox.create();
+    });
+
+    beforeEach(() => {
+        databaseConfig.type = 'SQLITE';
+        databaseConfig.name = 'predator';
+        databaseConfig.username = 'username';
+        databaseConfig.password = 'password';
+
+        sequelizeModelStub = sandbox.stub();
+        sequelizeUpsertStub = sandbox.stub();
+        sequelizeDefineStub = sandbox.stub();
+        sequelizeGetStub = sandbox.stub();
+        sequelizeStub = sandbox.stub();
+        sequelizeStub = sandbox.stub();
+        sequelizeGeValueetStub = sandbox.stub();
+
+        sequelizeDefineStub.returns({
+            hasMany: () => {
+            },
+            sync: () => {
+            }
+        });
+
+        sequelizeModelStub.returns({
+            key: {},
+            value: {},
+            findAll: sequelizeGetStub,
+            find: sequelizeGeValueetStub,
+            upsert: sequelizeUpsertStub
+        });
+
+        sequelizeStub.returns({
+            model: sequelizeModelStub,
+            define: sequelizeDefineStub
+        });
+        sequelizeStub.DataTypes = {};
```

Comment:
you dont need to put mode, define inside sequelizeStub... 
just pass them inside object 
{
 model: sequelizeModelStub,
            define: sequelizeDefineStub
}